integration-tests/suites/base.go

```diff
@@ -69,13 +69,26 @@ func (s *IntegrationTestSuiteBase) StartCollector(disableGRPC bool, options *com
 
 	s.Require().NoError(s.Collector().Setup(options))
 	s.Require().NoError(s.Collector().Launch())
-	// Wait for collector to report healthy, includes initial setup and probes
-	// loading. It doesn't make sense to wait for very long, limit it to 1 min.
-	_, err := s.waitForContainerToBecomeHealthy(
-		"collector",
-		s.Collector().ContainerID,
-		defaultWaitTickSeconds, 1*time.Minute)
-	s.Require().NoError(err)
+
+	// Verify if the image we test has a health check. There are some CI
+	// configurations, where it's not the case. If something went wrong and we
+	// get an error, treat it as no health check was found for the sake of
+	// robustness.
+	hasHealthCheck, err := s.findContainerHealthCheck("collector",
+		s.Collector().ContainerID)
+
+	if hasHealthCheck && err == nil {
+		// Wait for collector to report healthy, includes initial setup and
+		// probes loading. It doesn't make sense to wait for very long, limit
+		// it to 1 min.
+		_, err := s.waitForContainerToBecomeHealthy(
+			"collector",
+			s.Collector().ContainerID,
+			defaultWaitTickSeconds, 1*time.Minute)
+		s.Require().NoError(err)
+	} else {
+		fmt.Println("No HealthCheck found, do not wait for collector to become healthy")
+	}
 
 	// wait for self-check process to guarantee collector is started
 	selfCheckOk := s.Sensor().WaitProcessesN(s.Collector().ContainerID, 30*time.Second, 1)
```

Comment:
```suggestion
	selfCheckOk := s.Sensor().WaitProcessesN(s.Collector().ContainerID, 180*time.Second, 1)
```
I would be curious whether this fixes the issue.